University-advisor-web/Models/UserModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace University_advisor_web.Models
+{
+    public class UserModel
+    {
+        public UserModel()
+        {
+            Universities = new List<SelectListItem>
+            {
+                // TODO: Needs to be fetched from DB
+                new SelectListItem("Vilnius university", "Vilnius university"),
+                new SelectListItem("ISM", "ISM"),
+                new SelectListItem("Vilnius Gediminas Technical University", "Vilnius Gediminas Technical University")
+            };
+            Statuses = new List<SelectListItem>
+            {
+                new SelectListItem("Student", "Student"),
+                new SelectListItem("Lecturer", "Lecturer"),
+                new SelectListItem("Graduate", "Graduate")
+            };
+        }
```

Comment:
I think it may be better to have seperate University and Status models and then call those in UserModel. Since we will need to access Universities from settings and reviews aswell.

Reply:
I guess it would be better to have a RegistrationFormModel (that will be used in registration form only),  UserModel (that could be used in anywhere in our app). Because I have a strong feeling that creating separate model for universities and status is useless.